api/PclusterApiHandler.py

```diff
@@ -663,9 +663,64 @@ def _get_params(_request):
     params.pop("path")
     return params
 
+# Check if parallelcluster tags are active
+def check_tags(self, inactive_tags):
+    try:
+        cost_explorer = boto3.client('ce',)
+        tags = ['parallelcluster:attributes', 
```

Comment:
Let's pull these tags out into a global variable defined at the top and re-use that in both functions.